jsapp/js/components/modalForms/submission.es6

```diff
@@ -154,22 +155,37 @@ class Submission extends React.Component {
     dialog.set(opts).show();
   }
 
+  onDeletedSubmissionCompleted() {
+    stores.pageState.hideModal();
+  }
+
   launchEditSubmission() {
     this.setState({
       promptRefresh: true,
-      isEditLoading: true
+      isEditLoading: true,
+      isEdittingDuplicate: true
     });
-    enketoHandler.editSubmission(this.props.asset.uid, this.props.sid).then(
+    enketoHandler.editSubmission(this.props.asset.uid, this.state.sid).then(
       () => {this.setState({isEditLoading: false});},
       () => {this.setState({isEditLoading: false});}
     );
   }
 
+  duplicateSubmission() {
+    // Due to how modals are created, we must close this modal and recreate
+    // an almost identical one to display the new submission with a different
+    // title bar
+    stores.pageState.hideModal();
+    actions.resources.duplicateSubmission(this.props.asset.uid, this.state.sid, this.state.submission);
+  }
+
   triggerRefresh() {
     this.getSubmission(this.props.asset.uid, this.props.sid);
     this.setState({
       promptRefresh: false
     });
+    // Empty action listener to prompt table to refresh submission list
```

Comment:
To be clear: it's not empty, it just doesn't call `dataInterface` inside, as we usually do, but it's a 100% valid action :)